full_install.sh

```diff
@@ -3,6 +3,7 @@ set -euo pipefail
 python3 -m pip install -e core/
 python3 -m pip install -e features/
 python3 -m pip install -e tabular/[all]
+python3 -m pip install -e forecasting/[all]
```

Comment:
why do you need [all] option here? Are you planning to add optional dependencies for forecasting?

Reply:
What is this [all] for? I had it just because I was following the tabular.

Reply:
It's to also install optional dependencies as well as required dependencies. Let's remove it for now since there are no optional dependencies.

Reply:
You should remove the [all] for the forecasting install